src/test/resources/static/tree.html

```diff
@@ -0,0 +1,122 @@
+<!--
+ Copyright (C) 2019 Toshiba Corporation
+ SPDX-License-Identifier: Apache-2.0
+-->
+
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<style>
+		html,
+		body {
+			width: 100%;
+			height: 100%;
+			margin: 0;
+			padding: 0;
+			background-color: #eeeeee;
+		}
+
+	</style>
+</head>
+
+<body>
+	<script src="../META-INF/resources/webjars/pixi/latest/pixi.js"></script>
+	<script src="../META-INF/resources/webjars/wcardinal-ui/latest/wcardinal-ui.min.js"></script>
+	<script src="../META-INF/resources/webjars/wcardinal-ui/latest/wcardinal-ui-theme-white.min.js"></script>
```

Comment:
"../META-INF/resources" is not necessary. Please change to "/webjars/wcardinal-ui/wcardinal-ui-theme-white.min.js".